### ISSUE TITLE
Set Collider Transform assumes that mBody is a RigidBody

### ISSUE THREAD

**Issue by AlexanderGhosh** [DanielChappuis/reactphysics3d#362]
In the function `Collider::setLocalToBodyTransform` it makes the assumption that 'mBody' is a RigidBody

```c++
RigidBody* rigidBody = static_cast<RigidBody*>(mBody);
if (rigidBody != nullptr) {
    rigidBody->setIsSleeping(false);
}
``` 

however this assumption doesn't not hold when the Collider is attached to a CollisionBody causing an assertion to fail when calling `RigidBody::setIsSleeping`

**Comment by DanielChappuis** [comment]
Thanks for reporting this issue. This is already fixed in the 'develop' branch and will be part of the next release of the library.

**Comment by DanielChappuis** [comment]
This is now fixed in version [v0.10.0](https://github.com/DanielChappuis/reactphysics3d/releases/tag/v0.10.0) of the library. Thanks a lot for reporting the issue.
